### DISCUSSION TITLE
We lost site access!

### DISCUSSION THREAD

**Discussion by ArcerionDev** [Prodigy-Hacking/ProdigyMathGameHacking#1005] [(D) Developer]
In case you all didn't know, in order to run the hacks, you're actually using a modified version of the game itself, in order to be able to edit the player's properties and whatnot.

So, that domain used to be https://prodigyhacking.ml, but we lost access to it, so loading in an older version of the extension (v.2.1.4 or before) just won't work.

There's a security risk, however.

You're evaluating some random text from a random website, which means that we could run whatever we want on your device. 

However, as we are decent people here at the Prodigy Hacking organization, we don't want anything to do with your personal info, and we'll never run anything malicious on your device.
But as we lost the domain, someone else could buy it and steal all of your information, including your location, credentials, and basically anything as soon as you log into the game.

For this reason, please update to v2.1.5 as soon as possible.

Thank you all for your understanding.

~Arc

**Comment by Bruceq3456** [comment]
How do you update?

**Reply by Bruceq3456** [reply]
oh,nvm...The chrome store has the 2.1.5 extension,install that.

**Comment by ghost** [comment]
thank you for the suggestion, i will buy that domain

**Reply by ghost** [reply]
lol

**Comment by 29wumya** [comment]
That's bad,  Ok, I will update

**Comment by ShawnC32** [comment]
Here is another relaxing song
https://onedrive.live.com/?cid=632A712E352593DA&id=632A712E352593DA%21418&parId=632A712E352593DA%21413&o=OneUp

**Comment by ShawnC32** [comment]
In case you all didn't know, in order to run the hacks, you're actually using a modified version of the game itself, in order to be able to edit the player's properties and whatnot.

So, that domain used to be https://prodigyhacking.ml, but we lost access to it, so loading in an older version of the extension (v.2.1.4 or before) just won't work.

There's a security risk, however.

You're evaluating some random text from a random website, which means that we could run whatever we want on your device.

However, as we are decent people here at the Prodigy Hacking organization, we don't want anything to do with your personal info, and we'll never run anything malicious on your device.
But as we lost the domain, someone else could buy it and steal all of your information, including your location, credentials, and basically anything as soon as you log into the game.

For this reason, please update to v2.1.5 as soon as possible.

Thank you all for your understanding.

~Arc meri ass

**Reply by donaldli2020** [reply]
Please stop spamming.

**Reply by ShawnC32** [reply]
> @donaldli2020 He's not spamming @ShawnC32 You copied @ArcerionDev's text and added a swear at the end.

This excuse died

**Reply by ShawnC32** [reply]
> Please stop spamming.

This excuse is broken

**Comment by ShawnC32** [comment]
https://onedrive.live.com/?cid=632A712E352593DA&id=632A712E352593DA%213185&parId=632A712E352593DA%21413&o=OneUp
Anyone want to listen to relaxing music to calm down over problems with the phex?
